data/categories.txt

```diff
@@ -9893,6 +9893,46 @@ vi:Nhà máy bia
 zh-Hans:酿酒厂
 zh-Hant:釀酒廠
 
+craft-caterer
+en:Caterer
+ar:متعهد تقديم الطعام
+be:Кейтэрынг
+bg:Кетъринг
+ca:Càtering
+cs:Cateringová společnost
+da:Caterer
+de:Caterer
+el:Caterer
```

Comment:
This one is not translated, maybe some others too. Please use `tools/python/translate.py`

```
ar:متعهد تقديم الطعام
be:Арганізатар грамадскага харчавання
bg:Кетъринг
ca:Càtering
cs:Cateringová společnost
da:Caterer
de:Caterer
el:Caterer
es:Servicio de catering
es-MX:Abastecedor
et:Catering
eu:Catering
fa:پذیرایی
fi:Pitopalvelu
fr:Traiteur
he:קייטרינג
hi:भोजनादि का व्यवस्थापक
hu:Catering
id:Katering
it:Catering
ja:ケータリング
ko:케이터링 업체
mr:केटरर
nb:Caterer
nl:Traiteur
pl:Firma cateringowa
pt:Serviço de catering
pt-BR:Serviço de bufê
ro:Catering
ru:Кейтеринг
sk:Cateringová spoločnosť
sv:Cateringfirma
sw:Mpishi
th:พนักงานจัดเลี้ยง
tr:Yemek şirketi
uk:Постачальник продуктів харчування
vi:Người cung cấp thực phẩm
zh-Hans:餐饮
zh-Hant:餐飲服務商
```

Reply:
Yes, I did all translations using tools/python/translate.py. It seems it didn't work well for greek in this case. Fixed.